tests/manual_test_mcd.py

```diff
@@ -17,53 +17,61 @@
 
 import os
 import sys
-from pprint import pprint
 from web3 import Web3, HTTPProvider
 
 from pymaker import Address
-from pymaker.auctions import Flipper, Flapper, Flopper
 from pymaker.deployment import DssDeployment
 from pymaker.keys import register_keys
 from pymaker.numeric import Wad
 
-endpoint_uri = f"{os.environ['SERVER_ETH_RPC_HOST']}:{os.environ['SERVER_ETH_RPC_PORT']}"
-web3 = Web3(HTTPProvider(endpoint_uri=endpoint_uri, request_kwargs={"timeout": 10}))
+web3 = Web3(HTTPProvider(endpoint_uri=os.environ['ETH_RPC_URL'], request_kwargs={"timeout": 10}))
 web3.eth.defaultAccount = sys.argv[1]   # ex: 0x0000000000000000000000000000000aBcdef123
-register_keys(web3, [sys.argv[2]])      # ex: key_file=~keys/default-account.json,pass_file=~keys/default-account.pass
-
+if len(sys.argv) > 2:
```

Comment:
should this be: `len(sys.argv) > 1`?

Reply:
The idea is if the pass the key (2 arguments, length 3) it'll run transactions.  If they only pass the account address (1 argument, length 2) it'll just show balances.